meaningfulconsent/templates/base.html

```diff
@@ -226,13 +224,43 @@ <h4 class="modal-title" id="loginLabel">Facilitators</h4>
 	{% endif %}
 	
 	<script type="text/javascript">
-	    jQuery(document).ready(function() {
-		   // setup some ajax progress indicator
-		   jQuery("html").bind("ajaxStart", function(){  
-			  jQuery(this).addClass("busy");
-		   }).bind("ajaxStop", function() {  
-			  jQuery(this).removeClass("busy");
-		   });
+        function getCookie(name) {
+            var cookieValue = null;
+            if (document.cookie && document.cookie != '') {
+                var cookies = document.cookie.split(';');
+                for (var i = 0; i < cookies.length; i++) {
+                    var cookie = jQuery.trim(cookies[i]);
+                    // Does this cookie string begin with the name we want?
+                    if (cookie.substring(0, name.length + 1) == (name + '=')) {
+                        cookieValue = decodeURIComponent(cookie.substring(name.length + 1));
+                        break;
+                    }
+                }
+            }
+            return cookieValue;
+        }
+
+        jQuery(document).ready(function() {
+		    // setup some ajax progress indicator
+            jQuery("html").bind("ajaxStart", function() {  
+			    jQuery(this).addClass("busy");
+			}).bind("ajaxStop", function() {  
```

Comment:
I made the same mistake in one of my recent DMT pull requests. The tabs were probably already there.. it happens in the templates a lot since the designers are less prone to pay attention to invisible details like this in whatever editor they're using.